docs/source/nlp/usage_guides/nlp_properties_extended.rst

```diff
@@ -0,0 +1,77 @@
+.. _nlp__properties_ext:
+
+==========
+Properties
+==========
+
+Properties are one-dimension values that are extracted from the text. For example, a property could be as simple as
+**text length** or **sentiment**, or more complex properties such as the probability that the LLM avoided answering
+a question.
+
+Link Validly
+------------
+
+The Link Validity property represents the ratio of number links in the text that are valid links, divided by the total
+number of links. A valid link is a link that returns a 200 (OK) code when requested. For text without links, the
+property will always return 1 (all links valid).
+
+Readability Score
+-----------------
+
+A score calculated based on the
+`Flesch reading-ease <https://en.wikipedia.org/wiki/Flesch%E2%80%93Kincaid_readability_tests#Flesch_reading_ease>`_,
+calculated for each text sample. The score typically ranges from 0
+(very hard to read, requires intense concentration) to 100 (very easy to read).
```

Comment:
you say typically, but technically the max and min are different, right? Maybe worth mentioning